x-pack/plugins/event_log/server/plugin.ts

```diff
@@ -125,8 +125,7 @@ export class Plugin implements CorePlugin<IEventLogService, IEventLogClientServi
   > => {
     return async (context, request) => {
       return {
-        getEventLogClient: () =>
-          this.eventLogClientService!.getClient(request, context.core.savedObjects.client),
+        getEventLogClient: () => this.eventLogClientService!.getClient(request),
```

Comment:
Here's the fix for accessing event log for saved objects in non-default spaces.  The `getClient()` call already built a properly scoped SO client, we were passing one in here that didn't work, so just changed to use the one it built instead.